src/server/checkBrowserVersion.js

```diff
@@ -0,0 +1,37 @@
+const childProcess = require('child_process');
+
+const firefox = require('selenium-webdriver/firefox');
+
+const config = require('./config');
+
+const MINIMUM_FIREFOX_VERSION = 50.0;
+const FIREFOX_VERSION_MATCHER = /Mozilla Firefox ([0-9.]+)/;
```

Comment:
Alternatively, you could make the error message less useful by not including the current version. That would be a strict improvement over giving potentially incorrect information.